dlt-database/entity/0001-init_db/TransactionDraft.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  CreateDateColumn,
+  ManyToOne,
+  OneToOne,
+  JoinColumn,
+} from 'typeorm'
+import { Decimal } from 'decimal.js-light'
+
+import { DecimalTransformer } from '../../src/typeorm/DecimalTransformer'
+import { Account } from './Account'
+import { Community } from './Community'
+import { ConfirmedTransaction } from './ConfirmedTransaction'
+
+@Entity('transaction_drafts')
+export class TransactionDraft {
+  @PrimaryGeneratedColumn('increment', { unsigned: true, type: 'bigint' })
+  id: number
+
+  @Column({ name: 'iota_message_id', type: 'binary', length: 32, nullable: true })
+  iotaMessageId?: Buffer
+
+  @ManyToOne(() => Account, (account) => account.transactionDraftsSigning)
+  @JoinColumn({ name: 'signing_account_id' })
+  signingAccount: Account
+
+  @Column({ name: 'signing_account_id', type: 'int', unsigned: true })
+  signingAccountId: number
+
+  @ManyToOne(() => Account, (account) => account.transactionDraftsRecipient)
+  @JoinColumn({ name: 'recipient_account_id' })
+  recipientAccount?: Account
+
+  @Column({ name: 'recipient_account_id', type: 'int', unsigned: true, nullable: true })
+  recipientAccountId?: number
```

Comment:
Yes.
https://dict.leo.org/forum/viewGeneraldiscussion.php?idThread=1313770&lang=en
ChatGPT also advises me to use recipient instead of receiver. We had the discussion in the team a long time ago, because I used to use receiver.